src/views/clusteroverview/overview/components/details-card/SourceMissingStatus/WarningStatus.tsx

```diff
@@ -0,0 +1,19 @@
+import * as React from 'react';
+
+import { useKubevirtTranslation } from '@kubevirt-utils/hooks/useKubevirtTranslation';
+import {
+  GenericStatus,
+  StatusComponentProps,
+  YellowExclamationTriangleIcon,
+} from '@openshift-console/dynamic-plugin-sdk';
+
+const WarningStatus: React.FC<StatusComponentProps> = (props) => {
+  const { t } = useKubevirtTranslation();
+  const { title } = props;
+
+  return (
+    <GenericStatus {...props} Icon={YellowExclamationTriangleIcon} title={title || t('Warning')} />
```

Comment:
I can't see an actual reason to add `{...props}` from this PR, will it have a future purpose?

Reply:
It can take other props defined in `StatusComponentProps`. The only reason `title` is destructured is so that an alternative can be set if it's empty. The rest are passed through as-is. I think it would be limiting to not pass through the rest of the props.